gammapy/maps/geom.py

```diff
@@ -940,33 +940,20 @@ def center_skydir(self):
         pass
 
     @classmethod
-    def read(cls, filename, **kwargs):
-        """Create a geometry object from a FITS file.
+    def from_hdulist(cls, hdulist, hdu=None, hdu_bands=None):
```

Comment:
@adonath - Please fix the docstring (I think the issue is that the directive name is `code-block`):
https://travis-ci.org/gammapy/gammapy/jobs/419742977#L1997

Also: you state that you give an example that doesn't read from file, but then do `fits.open(filename)`, i.e. read it from file.

I would suggest to just remove the example here, and then later extend the high-level docs section on creating and reading maps and geoms. Examples that can be copy & pasted and actually run are best.

Reply:
Addressed directly in master https://github.com/gammapy/gammapy/commit/9773edb507cec86d3acef9674ab9c8ca8efc4de5.